db/column_family.cc

```diff
@@ -929,8 +934,9 @@ bool ColumnFamilyData::NeedsCompaction() const {
 
 Compaction* ColumnFamilyData::PickCompaction(
     const MutableCFOptions& mutable_options, LogBuffer* log_buffer) {
+  SequenceNumber oldest_mem_seqno = super_version_->GetSmallestMemSeqno();
```

Comment:
Nit: now we have 3 terms in play: `earliest`, `smallest`, and `oldest`. For consistency, I think this should be `earliest_mem_seqno`.

Reply:
Ok， I'll fix it.